src/app/elastic/elastic.py

```diff
@@ -47,7 +47,8 @@ def query_materials(ancestor_id: UUID = None) -> Query:
 
 
 def query_missing_material_license() -> Query:
-    qfield = LearningMaterialAttribute.LICENSES
+    # use keyword field for exact match, no token matching
```

Comment:
Erm, please have a look at the handle_text_field function, which is deeply hidden in the code (to be refactored):

`
def handle_text_field(qfield: Union[ElasticField, str]) -> str:
    if isinstance(qfield, ElasticField):
        qfield_key = qfield.path
        if qfield.field_type is ElasticFieldType.TEXT:
            qfield_key = f"{qfield_key}.keyword"
        return qfield_key
    else:
        return qfield
`
License was already postfixed with "keyword". See furthermore `tests/unit_tests/crud/test_score.py`, line 105.

Basically, this change does nothing.

I agree with Martin, this will be further refactored, e.g. Hexagonal Architecture, in the future